pkg/registry/apps/statefulset/strategy_test.go

```diff
@@ -227,43 +226,6 @@ func TestStatefulSetStrategy(t *testing.T) {
 			t.Errorf("expected PersistentVolumeClaimRetentionPolicy to be updated: %v", errs)
 		}
 	})
-	t.Run("when StatefulSetAutoDeletePVC feature gate is disabled, PersistentVolumeClaimRetentionPolicy should not be updated", func(t *testing.T) {
-		featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, features.StatefulSetAutoDeletePVC, true)
```

Comment:
Is it a bad test? The test name is `StatefulSetAutoDeletePVC feature gate is disabled`, but the feature-gate is set to true.

Reply:
/cc @mattcary